content/en/account_management/saml/mobile-idp-login.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Logging into the Mobile App when using IdP Initiated SAML
+kind: documentation
+aliases:
+  - /account_management/faq/how-do-i-use-the-mobile-app-with-saml/
+further_reading:
+- link: "/account_management/saml/"
+  tag: "Documentation"
+  text: "Configure SAML for your Datadog account"
+- link: "/account_management/multi_organization/"
+  tag: "Documentation"
+  text: "Configuring Teams & Organizations with Multiple Accounts"
+---
+
+## Using the Datadog Mobile App with your IdP Initiated SAML (SSO) Provider
+<div class="alert alert-warning">
+Logging into the Mobile App with IdP Initiated SAML is an opt-in feature. Reach out to Support to have the feature enabled for your account before you alter your SAML configuration.
+</div>
+
+In order to use the Datadog Mobile App with Identity Provider Initiated SAML you will need to pass an additional Relay State to Datadog in order to trigger the Mobile App landing page on login. When enabled all sign ins from SAML for that particular app will land on a page that page before proceeding. On Mobile devices with the Datadog Mobile App installed the App will capture the request and continue to sign in from the App.
```

Comment:
```suggestion
In order to use the Datadog Mobile App with Identity Provider (IdP) Initiated SAML, you need to pass an additional Relay State to Datadog in order to trigger the mobile app landing page on login. Once enabled, all sign ins from SAML for that particular app land on a page that page before proceeding. On mobile devices with the Datadog mobile app installed, the app will capture the request and continue to sign in from the app.
```

Reply:
`all sign ins from SAML for that particular app land on a page that page before proceeding` - the end of this sentence seems like it's duplicated, do we mean that they land on a page before proceeding?